### DISCUSSION TITLE
Cypress Soft assertion: When can we expect the Cypress soft assertion "npm i -D cypress-soft-assertions" support in cypress 13+ version?

### DISCUSSION THREAD

**Discussion by Aditya2680** [cypress-io/cypress#28968] [Questions and Help]
When can we expect the Cypress soft assertion support using plugin "npm i -D cypress-soft-assertions" in cypress 13+ version?

**Comment by portyu9** [comment]
Hi @Aditya2680 

`cypress-soft-assertions` is a **third-party plugin**, not a Cypress package, so support for Cypress 13+ would need to come from that plugin's maintainer rather than from a Cypress release.

The plugin's own README currently states:

- it does not support retries
- it works with Cypress v11

So there is no Cypress configuration option that will make that package supported on Cypress 13+.

Cypress itself also does not currently have a direct built-in equivalent to that plugin's soft-assertion behavior.

If soft assertions are required, the practical options are to use/fork a maintained implementation that explicitly supports your Cypress version, or restructure the assertions so failures are collected by application/test code and reported at the end.

Please mark this answer as accepted if it helped, thank you.
